crates/holochain/src/conductor/chc.rs

```diff
@@ -394,9 +395,10 @@ mod tests {
 
         dbg!(&hash1);
 
-        regex::Regex::new(
-            r#".*ChcHeadMoved\("SourceChain::flush", InvalidChain\((\d+), ActionHash\([a-zA-Z0-9-_]+\).*"#
-        ).unwrap().captures(&format!("{:?}", hash1)).unwrap();
+        regex::Regex::new(r#".*ChcError\(InvalidChain\((\d+), ActionHash\([a-zA-Z0-9-_]+\).*"#)
+            .unwrap()
+            .captures(&format!("{:?}", hash1))
+            .unwrap();
         // TODO: check sequence and hash
```

Comment:
Address todo?

Reply:
Only really feasible with proper matching. OK to not do this this time?